infrastructure/parameters/dev.parameters.json

```diff
@@ -15,6 +15,15 @@
       "value": "S2 Standard"
     },
     "skuImporter": {
+      "value": "GeneralPurpose GP_S_Gen5_1"
```

Comment:
Rather than using strings that you need to split up, couldn't we have these typesof as an array value like?

```
"value": ["GeneralPurpose", "GP_S_Gen5_1"]
```

Reply:
Yep good idea - will just see if the serverless works first & then do as you advise.